### PR TITLE
delete works

### PR DESCRIPTION
**Pull request by jsmnwltrs**

### CODE REVIEW

File: src/Components/Pages/Events/Events.js [hunk 1]
```diff
@@ -37,12 +37,28 @@ class Events extends React.Component {
...
+      .catch(error => console.error('error on deleteEvent', error));
+  }
+
   render() {
     const { events } = this.state;
     const eventItemComponents = events.map(event => (
       <EventItem
         event={event}
         key={event.id}
+        deleteSingleEvent={this.deleteEvent}s
```
Comment (morecallan) [inline src/Components/Pages/Events/Events.js]: Typo 😸

**Review by morecallan — COMMENTED** [review]
One small thing then I'll merge!

File: src/Components/Pages/Events/Events.js (hunk 1 above)
Comment (jsmnwltrs) [reply src/Components/Pages/Events/Events.js]: thanks! it should be gone now

**Comment by morecallan** [comment]
👍

**Pull request merged**